emily/handler/src/api/handlers/withdrawal.rs

```diff
@@ -79,18 +92,20 @@ pub fn create_withdrawal(
     operation_id = "updateWithdrawals",
     path = "/withdrawal",
     tag = "withdrawal",
+    request_body = UpdateWithdrawalsRequestBody,
     responses(
-        // TODO(271): Add success body.
-        (status = 201, description = "Withdrawals updated successfully", body = serde_json::Value),
+        (status = 201, description = "Withdrawals updated successfully", body = UpdateWithdrawalsResponse),
         (status = 400, description = "Invalid request body"),
         (status = 404, description = "Address not found"),
         (status = 405, description = "Method not allowed"),
         (status = 500, description = "Internal server error")
     )
 )]
 pub fn update_withdrawals(
-    _body: serde_json::Value,
-    path: FullPath,
+    _body: UpdateWithdrawalsRequestBody,
```

Comment:
CI fails here. I guess this should be `WithdrawalUpdate`.

Reply:
I changed it last minute to `WithdrawalUpdateRequestBody` and missed this one! Thanks!